AGPs/AGP-CRDAO.md

```diff
@@ -0,0 +1,60 @@
+---
+AGP: N/A
+Title:  Community Rewards DAO
+Author: Aaron Foster (`@pythonpete32`) & Burrrata (`@burrrata`)
+Status: Stage III
+Track: Finance
+Created: 2019-10-01
+---
+
+# AGP X: Community Rewards DAO
+
+## Address of the transfer recipient
+
+TBD, but will be the Vault of the CRDAO
+
+## Amount of the transfer
+
+12,000 ANT
+- 1000 ANT for every week between this upcoming AGP (fourth Thursday of October) and the next AGP (fourth Thursday of January)
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+1
+
+## Purpose of the transfer
+
+The CRDAO will reward contributions to the Aragon ecosystem with small amounts of ANT. Its purpose is to reward people for contributions they do and encourage them to do more voluntarily without necessarily an expectation of specific remuneration. it differs from CFDAO in that recipients do not apply for funds, CRDAO rewards are recognition for service to the Aragon Community.
+
+CRDAO will have a fixed weekly budget. This will provide a runway from this AGP to the next. The budget will be divided amongst nominated Aragon community members using Autarks Dot Voting app. The reward policy will be based on the [1Hive model](https://1hive.org/contribute/allocations-payroll), however a key difference being that any member of the Aragon forum can nominate anyone else, but it is up to the DAO members to actually vote on those nominations. 
```

Comment:
Autark's Projects app is still not live on mainnet and it is unclear whether it will be right after this ANV. Would this be a blocker? How would the ANT be allocated if Projects is not live?

Reply:
Updated proposal to address this